_topics/en/client-side-validation.md

```diff
@@ -56,27 +56,35 @@ excerpt: >
   in the validation.
 
 ---
-A conceptually simple client-side validation protocol might associate a
-token with a particular UTXO.  Only the set of validators needs to know
-about that association; it does not need to be published to the block
-chain or anywhere else that is public.  When the UTXO is spent, the
-spending transaction associates the token with a new UTXO.  If Alice
+A conceptually simple client-side validation protocol might assign an
+off-chain state (like a number of tokens) with a particular UTXO.
```

Comment:
```suggestion
off-chain state (like a set of tokens) with a particular UTXO.
```

Reply:
Nope, it is about the amount of tokens owned by a person

Reply:
```suggestion
off-chain state (like an amount of owned tokens) with a particular UTXO.
```

Reply:
Or perhaps "like the number of owned tokens"?

Google search brought up this from an English writing blog:
> Though number and amount have similar meanings, number is used for things that are countable, while amount is used for things that are uncountable. Knowing when to use either is a matter of looking at the noun that's being described.

Reply:
Bitcoin Core and other libraries use amount instead of number, so I assume amount is more common in this context

Reply:
I guess I'm unsure of the concept being conveyed. In bitcoin, _amount_ is used for the _value_ of a transaction output (or sum of several outputs), in units of satoshis (or often converted to BTC). I'm not sure what a token is here, but if what's meant is the **_number_** of tokens (like 1, 2, 3, etc.), then _number_ is the correct usage.

Reply:
In RGB everywhere it is called "amount of tokens" - the value assigned to a specific UTXO or put into the invoice. Even if the choice of the word is suboptimal, it is the word used everywhere in the code, in the docs, in the talks etc.